### ISSUE TITLE
Reasons in finalised stocktake should show modal when pressed

### ISSUE THREAD

**Issue by Chris-Petty** [openmsupply/mobile#1010]
**Build Number:** 2.3.3

**Description:**
In a finalised stocktakes the reasons are shown. They get truncated with ellipses, then you can't read them. Also they are still touchable but don't do anything when pressed.

![image](https://user-images.githubusercontent.com/7684221/58516655-d4983f80-81fc-11e9-8504-e9b238f716df.png)

**Expected:**
That a dialog would show that displays the full reason. No editing or anything. At the very least if it doesn't do anything, it shouldn't be a `TouchableOpacity` component.

**Reproducible:** Yes

**Reproduction Steps:**
Go to a finalised stocktake with reasons

**Comments:**

**Comment by joshxg** [comment]
What kind of dialog were you thinking? I am thinking just disable it, maybe increase the width of the column. Although I do like a popup, maybe the bottom modal similar to naming a stocktake?

Also bring this component over from vaccines .. (made for exactly this type of cell)

https://github.com/openmsupply/mobile/blob/vaccine-development/src/widgets/IconCell.js

**Comment by Chris-Petty** [comment]
@joshxg There isn't any limit on reason length, so increasing column width can only do so much.
I like bottom modal idea. 
Aren't you already using IconCell in stocktakes for the batches modal?

**Comment by joshxg** [comment]
@Chris-Petty no, it's a hacked together assortment of rubbish I only made it when we started vaccines, if I remember correctly
